content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -210,10 +210,22 @@ The initial imported plugins file can be
 
 ### Support file
 
-By default Cypress will automatically include the support file
-`cypress/support/index.js`. This file runs **before** every single spec file. We
-do this purely as a convenience mechanism so you don't have to import this file
-in every single one of your spec files.
+By default Cypress will automatically include the supportFile. This file runs
+**before** every single spec file. We do this purely as a convenience mechanism
+so you don't have to import this file in every single one of your spec files.
+
+<Alert type="info">
+
+<strong class="alert-header"><Icon name="cogs"></Icon> Support File Per Test
+Type</strong>
+
+Depending on which Cypress [test type](/guides/overview/choosing-testing-type)
```

Comment:
Open to feedback.changes here. I like the idea of this sort of call out and deeplink but I don't think this is valuable as-is.